modules/AIM/module/inc/AIM/aim_config.h

```diff
@@ -209,7 +209,7 @@ struct aim_pvs_s;
 
 
 #ifndef AIM_CONFIG_INCLUDE_DAEMONIZE
-#define AIM_CONFIG_INCLUDE_DAEMONIZE 0
+#define AIM_CONFIG_INCLUDE_DAEMONIZE 1
```

Comment:
Not really, but its already changed to 1 from 0 in master as a result of a previous pull request. I don't know why it thinks its a new change.